plugins/modules/cp4s/cp4s_query_incidents.py

```diff
@@ -0,0 +1,262 @@
+
+# Copyright: (c) 2021, Ryan Gordon
```

Comment:
Add your name to the copyright for this file. ;)

Reply:
thanks :)